src/replicache-options.ts

```diff
@@ -224,4 +224,9 @@ export type ReplicacheInternalOptions = {
    * Defaults to true.
    */
   enableMutationRecovery?: boolean;
+
+  /**
+   * Allows exposing _persist to perf test
+   */
+  exposePersistMethodAs?: symbol;
```

Comment:
@grgbkr This is another new "hack" to expose internals.

When we minimize we rename all properties that start with `_` so `_persist` gets renamed.

Instead we expose the method on this using a symbol which the subclass passes in.

Reply:
I don't love it.  I don't hate it.  I don't have a better suggestion.

Reply:
Something that may type check a bit better, and also can more easily expose additional internal apis.
```
interface ReplicacheInternalOptions {
  receiveInternalAPI?: (internalAPI: ReplicacheInternalAPI) => void;
}

interface ReplicacheInternalAPI {
  persist(): Promise<void>;
}
```